src/components/SearchResults.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+
+const SearchResults = ({ bookings }) => {
+  const [selectedCustomerId, setSelectedCustomerId] = useState(null);
+
+  const handleShowProfile = (customerId) => {
+    setSelectedCustomerId(customerId);
+  };
+
+  return (
+    <div>
+      <table className="table table-striped">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>First Name</th>
+            <th>Surname</th>
+            <th>Email</th>
+            <th>Room ID</th>
+            <th>Check-in Date</th>
+            <th>Check-out Date</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {bookings.map((booking) => (
+            <tr key={booking.id}>
+              <td>{booking.title}</td>
+              <td>{booking.firstName}</td>
+              <td>{booking.surname}</td>
+              <td>{booking.email}</td>
+              <td>{booking.roomId}</td>
+              <td>{booking.checkInDate}</td>
+              <td>{booking.checkOutDate}</td>
+              <td>
+                <button
+                  className="btn btn-primary"
+                  onClick={() => handleShowProfile(booking.id)}
+                >
+                  Show profile
+                </button>
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      {selectedCustomerId && (
+        <CustomerProfile customerId={selectedCustomerId} />
+      )}
+    </div>
+  );
+};
+
+const CustomerProfile = ({ customerId }) => {
```

Comment:
You used the Id really well here to allow for searching. Only suggestion would be to move this into a separate file so it becomes its own reusable component if it needs to be used elsewhere. Similar to the other components in the app.